templates/super_glue/record/templates.yaml

```
@@ -1,15 +1,38 @@
 dataset: super_glue
 subset: record
 templates:
+  014b669e-2e3b-40ce-bdde-418966c7d666: !Template
+    id: 014b669e-2e3b-40ce-bdde-418966c7d666
+    jinja: "{{ passage }} \n{{ query }} \nWhich one is the placeholder? {{ entities\
```

mapping to the exact token used in {{passage}}
```suggestion
    jinja: "{{ passage }} \n{{ query }} \nWhich one is the \"@placeholder\"? {{ entities\
```